src/storage/storageController.ts

```diff
@@ -24,52 +26,50 @@ export enum DefaultSavingLocations {
   'Session Only' = 'Session Only'
 }
 
-// A saved connection does not contain the connection information,
-// just metadata about the connection and an id for referencing secure storage.
-export type SavedConnection = {
-  id: string; // uuidv4
-  name: string; // Possibly user given name, not unique.
-  storageLocation: StorageScope;
+export type ConnectionsFromStorage = {
+  [key: string]: StoreConnectionInfo
```

Comment:
(nit:) Maybe this could be

```suggestion
  [connectionId: string]: StoreConnectionInfo
```

for clarity, if I’m following everything correctly